wiki/Skinning/Guides_and_important_threads/de.md

```diff
@@ -0,0 +1,31 @@
+# Tutorials zu Skinning und wichtige Threads
```

Comment:
```suggestion
# Anleitungen zu Skinning und wichtige Threads
```
Probably makes sense to change the heading, since you used `Anleitungen` further down as well (and i like it more than Tutorials :sweat_smile:)